mypy/test/data.py

```diff
@@ -689,6 +690,11 @@ def split_test_cases(
                     name, parent.name, line_no
                 )
             )
+        version = m.group("version")
+        if version:
+            if sys.version_info < (3, int(version.split(".")[1])):
```

Comment:
why not just compare againt the whole value
```suggestion
            if sys.version_info < (int(num) for num in version.split(".")):
```
or is there a function somewhere that already exists to compare versions as strings

Reply:
because it captures the `-3`